src/machines/java/com/enderio/machines/client/gui/screen/XPVacuumScreen.java

```diff
@@ -0,0 +1,65 @@
+package com.enderio.machines.client.gui.screen;
+
+import com.enderio.EnderIO;
+import com.enderio.base.common.lang.EIOLang;
+import com.enderio.core.client.gui.screen.EIOScreen;
+import com.enderio.core.client.gui.widgets.EnumIconWidget;
+import com.enderio.core.client.gui.widgets.ToggleImageButton;
+import com.enderio.core.common.util.Vector2i;
+import com.enderio.machines.client.gui.widget.FluidStackStaticWidget;
+import com.enderio.machines.common.menu.XPVacuumMenu;
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.gui.components.ImageButton;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Inventory;
+
+public class XPVacuumScreen extends EIOScreen<XPVacuumMenu> {
+
+    private static final ResourceLocation XP_VACUUM_BG = EnderIO.loc("textures/gui/xp_vacuum.png");
+    private static final ResourceLocation BUTTONS = EnderIO.loc("textures/gui/icons/buttons.png");
+    private static final ResourceLocation WIDGETS = EnderIO.loc("textures/gui/40/widgetsv2.png");
```

Comment:
widgetsv2 isn't used, extract the things you need from there to a seperate texture file